src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -972,25 +978,29 @@ public Parent validateParent(Parent parent, String chemist, Collection<BulkLoadP
 				}
 			}
 		}
-		// check for multiple fragments, and if isMixture or not
-		Boolean hasMultipleFragments = parent.getMultipleFragments();
-		if (hasMultipleFragments == null){
-			hasMultipleFragments = chemStructureService.checkForSalt(parent.getMolStructure());
-		}
-		if (hasMultipleFragments) {
-			// multiple fragments
-			if (parent.getIsMixture() != null) {
-				if (!parent.getIsMixture()) {
+
+		// Only run mixture check if we have not identified an already existing parent
+		if(parent.getId() != null) {
```

Comment:
I'm confused by your comment plus this line:
"Only run mixture check if we have not identified an already existing parent"
An existing parent would have id != null
So you go into the `if` clause only for pre-existing parents, and then run the multiple fragment check?

Reply:
Thanks. That was wrong. I fixed it.